### PR TITLE
Select multi tags

### PR DESCRIPTION
**Pull request by mfrachet**
# Select multi tags

## Description

Add a `withTags` props to the component

## Demo

Example on : https://design-system-git-select-multi-tags-strapijs.vercel.app/?path=/story/design-system-atoms-select--multi-with-tags

## API

```diff
<Select
    id="select1"
    label="Choose your meal"
    placeholder="Your example"
    onClear={() => setValues([])}
    hint="Description line"
    clearLabel="Clear the meal"
    error={error}
    value={values}
    onChange={onValueChange}
    disabled={disabled}
    customizeContent={(values) => `${values.length} currently selected`}
    multi
+  withTags
  >
    <Option value={'pizza'}>Pizza</Option>
    <Option value={'hamburger'}>Hamburger</Option>
    <Option value={'bagel'}>Bagel</Option>
</Select>
```

## Voiceover

Selecting pizza and hamburger in in listbox

### CODE REVIEW

**Review by soupette — APPROVED** [review]
LGTM

File: packages/strapi-design-system/src/Select/SelectTags.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  return (
+    <SelectTagsWrapper>
+      <Row wrap="wrap">
+        {tags.map((tag) => (
+          <SelectTag
+            icon={<CloseIcon />}
+            aria-disabled={disabled}
+            onClick={() => handleClick(tag.value)}
+            tabIndex={-1}
+            key={`tag-${tag.value}`}
```
Comment (HichamELBSI) [inline packages/strapi-design-system/src/Select/SelectTags.js:35]: this will work only if the tags are all different right?

**Review by HichamELBSI — CHANGES_REQUESTED** [review]
Good job @mfrachet! I just put two comments

File: packages/strapi-design-system/src/Select/SelectTags.js (hunk 1 above)
Comment (soupette) [reply packages/strapi-design-system/src/Select/SelectTags.js:35]: At some point they should be, if it happen to be problematic we can always make an update later to generate a unique id.

File: packages/strapi-design-system/src/Select/Select.stories.mdx [hunk 2]
```diff
@@ -91,3 +91,52 @@ Description...
...
+            label="Choose your meal"
+            placeholder="Your example"
+            onClear={() => setValues([])}
+            hint="Description line"
+            clearLabel="Clear the meal"
+            error={error}
+            value={values}
+            onChange={onValueChange}
+            disabled={disabled}
+            customizeContent={(values) => `${values.length} currently selected`}
```
[the remark this answers is not in the document]
Comment (mfrachet) [reply packages/strapi-design-system/src/Select/Select.stories.mdx:121]: yes

File: packages/strapi-design-system/src/Select/SelectTags.js (hunk 1 above)
Comment (mfrachet) [reply packages/strapi-design-system/src/Select/SelectTags.js:35]: What would be the value of having multiple time the same identifier in a list of options?  What is different from selecting an option "pizza" and an option "pizza"?

I personnally think they should be unique 😊

**Review by HichamELBSI — APPROVED** [review]
LGTM good job !!

**Pull request merged**